pkg/utils/embed/resources/validator/vsphere-root-level-privileges-all.yaml

```diff
@@ -1,431 +1,431 @@
-  - Cryptographer.Migrate
-  - VApp.InstanceConfig
-  - VirtualMachine.Config.CPUCount
-  - VirtualMachine.Config.AddNewDisk
-  - TrustedAdmin.RetrieveHostMetadata
-  - VirtualMachine.State.RevertToSnapshot
-  - Datastore.Browse
-  - VApp.ExtractOvfEnvironment
-  - HLM.Create
-  - ScheduledTask.Edit
-  - VirtualMachine.Config.ChangeTracking
-  - VirtualMachine.Interact.DefragmentAllDisks
-  - Profile.Clear
-  - VirtualMachine.Provisioning.ModifyCustSpecs
-  - VirtualMachine.Provisioning.CreateTemplateFromVM
-  - ContentLibrary.CreateSubscribedLibrary
-  - Namespaces.ListAccess
-  - Global.CapacityPlanning
-  - Alarm.ToggleEnableOnEntity
-  - Host.Local.CreateVM
-  - Authorization.ReassignRolePermissions
-  - EAM.View
-  - Global.VCServer
-  - Task.Create
-  - VirtualMachine.Inventory.Create
-  - AutoDeploy.Rule.Delete
-  - VcIntegrity.lifecycleSettings.Read
-  - VcIntegrity.Baseline.com.vmware.vcIntegrity.ManageBaselines
-  - Sessions.ImpersonateUser
-  - Alarm.SetStatus
-  - VirtualMachine.Interact.DnD
-  - VcIntegrity.lifecycleGeneral.Read
-  - VcIntegrity.Updates.com.vmware.vcIntegrity.Remediate
-  - Namespaces.ManageCapabilities
-  - Global.Settings
-  - vService.CreateDependency
-  - Datacenter.IpPoolConfig
-  - Global.GlobalTag
-  - VcIntegrity.FileUpload.com.vmware.vcIntegrity.ImportFile
-  - DVSwitch.Modify
-  - HealthUpdateProvider.Unregister
-  - VcIdentityProviders.Manage
-  - Resource.ColdMigrate
-  - TrustedAdmin.ManageTrustedHosts
-  - VcIntegrity.lifecycleDepots.Delete
-  - Resource.RenamePool
-  - Host.Inventory.ManageClusterLifecyle
-  - DVSwitch.Create
-  - VirtualMachine.Interact.CreateScreenshot
-  - VirtualMachine.State.RemoveSnapshot
-  - AutoDeploy.Profile.Create
-  - IntercomNamespace.Write
-  - ContentLibrary.UpdateLocalLibrary
-  - InventoryService.Tagging.EditCategory
-  - VirtualMachine.Provisioning.CloneTemplate
-  - VirtualMachine.Inventory.Register
-  - VApp.Move
-  - ServiceAccount.ManagePassword
-  - VirtualMachine.Interact.GuestControl
-  - InventoryService.Tagging.CreateCategory
-  - VApp.ManagedByConfig
-  - VirtualMachine.Provisioning.MarkAsTemplate
-  - Datacenter.IpPoolQueryAllocations
-  - VirtualMachine.Config.SwapPlacement
-  - VirtualMachine.Namespace.Event
-  - Resource.DeletePool
-  - ContentLibrary.PublishLibrary
-  - Datacenter.Create
-  - VApp.AssignResourcePool
-  - vSphereClient.UtilizeVerificationCode
-  - Host.Config.Settings
-  - Folder.Rename
-  - Cryptographer.Decrypt
-  - Extension.Register
-  - Cryptographer.AddDisk
-  - TrustedAdmin.ReadStsInfo
-  - VirtualMachine.Config.Settings
-  - VirtualMachine.Interact.DisableSecondary
-  - ContentLibrary.ProbeSubscription
-  - ContentLibrary.ReadStorage
-  - Host.Config.AuthenticationStore
-  - StorageViews.View
-  - Profile.View
-  - ContentLibrary.AddLibraryItem
-  - ContentLibrary.CreateLocalLibrary
-  - Resource.AssignVAppToPool
-  - VirtualMachine.GuestOperations.QueryAliases
-  - Host.Inventory.RemoveHostFromCluster
+- Alarm.Acknowledge
```

Comment:
thanks for putting this all in alphabetical order 😅 
going through this would be a pain otherwise

Reply:
np :)